client/elements/sc-page-dictionary.html

```diff
@@ -350,6 +357,11 @@ <h3>{{localize('similarSpelling')}}</h3>
               this.$.drawer.toggle();
           }
 
+          _createMetaData(dictionaryWord) {
+              this.metaData = {
+                'og:title': `Pali dictionary results for ${dictionaryWord}`
```

Comment:
can be an one-liner: `return { 'og:title': Pali dictionary results for ${dictionaryWord} }`